GridTariffApi.Lib.Tests/Services/V2/TariffQueryServiceTests.cs

```diff
@@ -36,6 +36,90 @@ private void Setup()
             _tariffQueryService = new TariffQueryService(null, null, _serviceHelper);
         }
 
+
+        [Fact]
+        public void AppendPowerPriceLevelsTest()
+        {
+            Setup();
+            var powerPrices = new Models.Digin.PowerPrices();
+
+            _tariffQueryService.AppendPowerPriceLevels(powerPrices, null,null,null);
+            Assert.Null(powerPrices.PriceLevels);
+
+            var powerPricePrices = new Models.PriceStructure.PowerPrices(null,null);
+            _tariffQueryService.AppendPowerPriceLevels(powerPrices, powerPricePrices, null, null);
+            Assert.Null(powerPrices.PriceLevels);
+
+            var powerPriceLevels = new List<PowerPriceLevel>
+            {
+                new PowerPriceLevel("a", null, null, null, null, null, 0, 0, null, null, null, null),
+                new PowerPriceLevel("b", null, null, null, null, null, 0, 0, null, null, null, null)
+            };
+
+            var daysInMonths = new List<int>();
+            daysInMonths.Add(28);
+            daysInMonths.Add(29);
+            daysInMonths.Add(30);
+            daysInMonths.Add(31);
+
+            var powerPricePricesWithLevels = new Models.PriceStructure.PowerPrices(powerPriceLevels, null);
+            var powerPrices2 = new Models.Digin.PowerPrices();
+            powerPrices2.PriceLevels = new Collection<Models.Digin.PowerPriceLevel>();
+            _tariffQueryService.AppendPowerPriceLevels(powerPrices2, powerPricePricesWithLevels, null, daysInMonths);
+
+            Assert.Equal(2, powerPrices2.PriceLevels.Count);
+            foreach (var priceLevel in powerPrices2.PriceLevels)
+            {
+                Guid guid = new Guid();
+                Assert.True(Guid.TryParse(priceLevel.Id, out guid));
```

Comment:
Det er vel kanskje ikke viktig at priceLevel.Id kan parses til Guid. Det er vel viktigere at de er unike id'er som ikke er samme verdi fra kall til kall. Så kanskje teste det i stedet?

Reply:
Dette er allerede implisitt testet pricelevelid i enhetstest ikke var satt opp med guid.
Har imidlertid endret dette slik at det både testes for guid-parseabilitiy, og for at returnert id ikke er samme som sendt inn.